streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -590,7 +570,7 @@ private void cleanGlobal(final boolean withIntermediateTopics,
             parameterList.add(resetScenarioArg);
         }
 
-        final String[] parameters = parameterList.toArray(new String[parameterList.size()]);
+        final String[] parameters = parameterList.toArray(new String[0]);
```

Comment:
```suggestion
        final String[] parameters = parameterList.toArray();
```

Reply:
If we're not going to pre-size the array, then giving it an empty array is equivalent to not passing in an array at all.

Reply:
We need to pass in the array to resolve the type. Otherwise the return type is `Object[]` instead of `String[]` and we would need to cast.